### PR TITLE
Update checkpoints votes UI earlier, tweak how we count votes

### PR DESCRIPTION
**Pull request by Doy-lee**
@jagerman

```
            // NOTE: Count checkpoints for other nodes, irrespective of being
            // a service node or not for statistics. Also count checkpoints
            // before the minimum lifetime for same purposes, note, we still
            // don't vote for the first 2 hours so this is purely cosmetic
```

### CODE REVIEW

File: src/cryptonote_core/service_node_quorum_cop.cpp [hunk 1]
```diff
@@ -212,8 +214,9 @@ namespace service_nodes
     service_nodes::quorum_type const max_quorum_type = service_nodes::max_quorum_type_for_hf(hf_version);
     bool tested_myself_once_per_block                = false;
 
-    time_t const now         = time(nullptr);
-    uint64_t const live_time = (now - m_core.get_start_time());
+    ptrdiff_t start_time      = m_core.get_start_time();
```
Comment (jagerman) [inline src/cryptonote_core/service_node_quorum_cop.cpp]: A `ptrdiff_t` is not guaranteed to be big enough to hold a time value, which could eventually be a problem here (versus using a time_t).

File: src/cryptonote_core/service_node_quorum_cop.cpp [hunk 2]
```diff
@@ -126,11 +126,13 @@ namespace service_nodes
...
-      int num_votes = 0;
+      int missed_votes = 0;
       for (checkpoint_vote_record const &record : proof.votes)
-        num_votes += record.voted;
+      {
+        if (!record.voted) missed_votes++;
+      }
 
-      if (num_votes <= CHECKPOINT_MAX_MISSABLE_VOTES)
+      if (missed_votes >= CHECKPOINT_MAX_MISSABLE_VOTES)
```
[the remark this answers is not in the document]
Comment (Doy-lee) [reply src/cryptonote_core/service_node_quorum_cop.cpp]: Yes, dun goofed.

File: src/cryptonote_core/service_node_quorum_cop.cpp [hunk 3]
```diff
@@ -212,8 +214,9 @@ namespace service_nodes
     service_nodes::quorum_type const max_quorum_type = service_nodes::max_quorum_type_for_hf(hf_version);
     bool tested_myself_once_per_block                = false;
 
-    time_t const now         = time(nullptr);
-    uint64_t const live_time = (now - m_core.get_start_time());
+    ptrdiff_t start_time      = m_core.get_start_time();
+    time_t const now          = time(nullptr);
+    ptrdiff_t const live_time = (now - start_time);
```
[the remark this answers is not in the document]
Comment (Doy-lee) [reply src/cryptonote_core/service_node_quorum_cop.cpp]: Sure, I'll take it. ptrdiff_t is a horrible name, I normally name it isize, or isize_t which is what it really it is.

File: src/cryptonote_core/service_node_quorum_cop.cpp (hunk 1 above)
Comment (Doy-lee) [reply src/cryptonote_core/service_node_quorum_cop.cpp]: Fixed

**Pull request merged**